DotNET/Complex Flow Examples/pdfa-3b-with-attachment.cs

```diff
@@ -0,0 +1,65 @@
+using Newtonsoft.Json.Linq;
+using System.Text;
+
+/* In this sample we will show how attach an xml document to a PDF file and then
+* convert the file with the attachment to conform to the PDF/A standard which
+* can be useful for invoicing and standards compliance. We will be running the
+*input document through /pdf-with-added-attachment to add the attachment and
+* then /pdfa to do the PDF/A conversion.
+
+* Note that there is nothing special about attaching an xml file, and any approprite
+* file may be attached and wrapped into the PDF/A conversion.
+*/
```

Comment:
Minor proofreading changes

```suggestion
/* In this sample, we will show how to attach an xml document to a PDF file and then
* convert the file with the attachment to conform to the PDF/A standard, which
* can be useful for invoicing and standards compliance. We will be running the
* input document through /pdf-with-added-attachment to add the attachment and
* then /pdfa to do the PDF/A conversion.

* Note that there is nothing special about attaching an xml file, and any appropriate
* file may be attached and wrapped into the PDF/A conversion.
*/
```